wbooks/src/app/components/Button/index.tsx

```diff
@@ -10,9 +10,9 @@ interface Props {
 
 const Information = ({ solid = true, text }: Props) => {
   return (
-    <TouchableOpacity style={solid ? styles.btnSolid : styles.btnHollow}>
+    <View style={solid ? styles.btnSolid : styles.btnHollow}>
```

Comment:
this not look good, it's a component of the all app named "Button" withOut touchable. I can see why you change this, but the component  name with your location not look good.

Reply:
If the component is named `Button` it is expected to have a 'pressable'  functionality

Reply:
Ok, I get it. I'm going to left the touchable with a dummy onPress